ivy/functional/ivy/general.py

```diff
@@ -2902,6 +2902,33 @@ def indices_where(
     ret
         Indices for where the boolean array is True.
 
+    Examples
+    --------
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([True, False, True, False, True])
+    >>> print(ivy.indices_where(x))
+    ivy.array([0, 2, 4])
+
+    >>> x = ivy.array([False])
+    >>> print(ivy.indices_where(x))
+    ivy.array([])
+
```

Comment:
Add examples with the out parameter being used as mentioned in the deep dive docs.